orangewidget/utils/combobox.py

```diff
@@ -238,6 +238,8 @@ def showPopup(self):
         popuprect_origin = style.subControlRect(
             QStyle.CC_ComboBox, opt, QStyle.SC_ComboBoxListBoxPopup, self
         )  # type: QRect
+        sl_margin = self.__searchline.style().pixelMetric(QStyle.PM_FocusFrameVMargin)
+        popuprect_origin.adjust(sl_margin/2 - 1, 0, -sl_margin*1.5 + 1, sl_margin)
```

Comment:
@irgolic, I get a perfect alignment if I remove `- 1` and `+ 1`. Could you check?

It would be nice to know where 1.5 comes from. It may be some other margin we need to take into account...

Reply:
You're right, I get perfect alignment without the `+/- 1` too. Go ahead and remove it.

No idea where the 1.5 comes in, I'm not sure why the box is so long without adjustments.